erpnext/vehicles/page/workshop_cp/workshop_cp.py

```diff
@@ -7,10 +7,16 @@
 	"vehicle_received_date",
 	"expected_delivery_date",
 	"name",
-	# "task_status",
+	# "tasks_status",
```

Comment:
Uncomment this ... now the field is allowed for sorting/filtering. Also update task status filter options